sw/device/tests/BUILD

```diff
@@ -1782,7 +1782,72 @@ opentitan_functest(
 opentitan_functest(
     name = "usbdev_test",
     srcs = ["usbdev_test.c"],
-    targets = ["verilator"],
+    targets = [
+        "cw310_rom",
+        "verilator",
+        "dv",
+    ],
+    verilator = verilator_params(
+        timeout = "long",
+    ),
+    deps = [
+        "//hw/top_earlgrey/sw/autogen:top_earlgrey",
+        "//sw/device/lib/dif:pinmux",
+        "//sw/device/lib/dif:usbdev",
+        "//sw/device/lib/runtime:log",
+        "//sw/device/lib/runtime:print",
+        "//sw/device/lib/testing:pinmux_testutils",
+        "//sw/device/lib/testing:usb_testutils",
+        "//sw/device/lib/testing:usb_testutils_simpleserial",
+        "//sw/device/lib/testing/test_framework:ottf_main",
+    ],
+)
+
+opentitan_functest(
+    name = "usbdev_stream_test_signed",
+    srcs = ["usbdev_stream_test.c"],
```

Comment:
Apologies, there's some confusion here resulting from a laptop change and someone trying to help me to re-establish the git connection/status for the repo on the new machine; documentation changes that are not related to usbdpi itself should not be in this commit (they were on a separate branch) and similarly usb20_agent.sv and sw/device/tests/BUILD were supposed to be excluded because usbdev_stream_test is intended for a later PR; I shall resolve this tonight/tomorrow.
Thanks, A